lib/round.rb

```diff
@@ -0,0 +1,56 @@
+class Round
+    attr_reader :deck,
+                :turns
+
+    def initialize(deck, turns = [])
+        @deck = deck
+        @turns = turns
+        @current_card_index = 0
+    end
+
+    def current_card
+        @deck.cards[@current_card_index]
+    end
+
+    def take_turn(guess)
+        card = current_card
+        @current_card_index += 1
+        new_turn = Turn.new(guess, card)
+        @turns << new_turn
+        return new_turn
+    end
```

Comment:
You could reorder lines 17 and 18, and then you don't need to save the current_card as `card`. 

```ruby
    def take_turn(guess)
        new_turn = Turn.new(guess, current_card)
        @current_card_index += 1
        @turns << new_turn
        return new_turn
    end

```